website/client/src/components/sidebarSection.vue

```diff
@@ -22,21 +22,21 @@
           :target="tooltipId"
         />
       </div>
-      <div
+      <button
```

Comment:
Looks like focused buttons get an outline 
<img width="345" alt="Schermata 2020-10-10 alle 10 59 12" src="https://user-images.githubusercontent.com/589911/95651053-b6786a00-0ae7-11eb-800a-debc985ccded.png">
<img width="200" alt="Schermata 2020-10-10 alle 10 59 51" src="https://user-images.githubusercontent.com/589911/95651063-bed0a500-0ae7-11eb-9cd1-5e67869782d9.png">

I'm wondering, given that browsers might apply their own styles to standard buttons if it would be better to use a div with `aria-role="button"`

@negue do you feel like this should be extracted into its own component?

Reply:
div with `aria-role` should be fine for that.

And yes since this button already has 2 different icons and a custom style it really waits to be its own component 😁

Reply:
I can set that up. Any naming conventions etc I should be aware of?

Reply:
Update: I've set it to a div but it still has the focus ring. I can get rid of it, but will still need to give it some sort of focus style for accessibility. What would work for you?

Reply:
@J-Sanderson the black ring as shown in my screenshot is fine, what i wanted to remove was the white outer ring

Reply:
How does this look?

![Screenshot from 2020-10-12 15-40-16](https://user-images.githubusercontent.com/17064002/95759272-6513da80-0ca1-11eb-9555-e2f48dd665f4.png)

Reply:
cc @Tressley

Reply:
This should be using focus states similar to the most recent Accessibility updates to the tasks page.

Reply:
purple borders? the styles that you see in the screenshot are from the browser itself and they're used everywhere we haven't customized them like the tasks page
<img width="172" alt="Schermata 2020-10-13 alle 09 59 11" src="https://user-images.githubusercontent.com/589911/95832667-e53a4e80-0d3a-11eb-919d-7e7dface8ab5.png">

Reply:
@paglias -- Ah, I could've been more clear in my initial reply. Yes, purple. Similar to the newest accessibility updates that were completed on the tasks controls.

<img src="https://user-images.githubusercontent.com/8680662/93027015-16e6bb00-f5bf-11ea-9e3a-6196e026ee86.gif">

Reply:
Thanks for the clarification!